test/e2e/antreapolicy_test.go

```diff
@@ -1764,6 +1766,59 @@ func testAppliedToPerRule(t *testing.T) {
 	executeTests(t, testCase)
 }
 
+func testACNPClusterGroupServiceRef(t *testing.T) {
```

Comment:
can we also test the affected pod add/update events too?

Reply:
Done